homeassistant/components/glances/sensor.py

```diff
@@ -290,7 +306,7 @@ def available(self) -> bool:
         return False
 
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> StateType | datetime:
```

Comment:
It would be better to use `_handle_coordinator_update()` also here so you can remove `native_value` completely as it's defined already in `SensorEntity`

Reply:
Thanks for the suggestion, I wanted to minimize change to the existing sensors but it makes sense to be more consistent with the uptime sensor.
I'll test the change and submit back if this works.

Reply:
Done, I have moved the code to `_handle_coordinator_update()` and removed the `native_value() `override